### DISCUSSION TITLE
AboutPage() cannot be deleted

### DISCUSSION THREAD

**Discussion by HenriquePF** [varabyte/kobweb#455] [General]
For some reason, i cannot delete the AboutPage(About.kt). And i don't know if that is the right directory that was generated. Found  nothing in the discussion about it.
<img width="453" alt="Screenshot 2024-01-31 at 13 14 41" src="https://github.com/varabyte/kobweb/assets/46723055/20874992-476b-4fb2-8847-a4b74883933b">

**Comment by bitspittle** [comment]
Search your project for an "about.md" file and delete that.

See also: https://github.com/varabyte/kobweb#markdown
